src/main/java/rx/Completable.java

```diff
@@ -1927,7 +1933,12 @@ public void onSubscribe(Subscription d) {
         
         return mad;
     }
-    
+
+    private static void deliverUncaughtException(Throwable e) {
```

Comment:
Well, we can create `ExceptionsInternal` under `internal` package, but yeah… k, let's extract it if somewhere else same functionality will be needed.